### ISSUE TITLE
Display "Help" key correctly

### ISSUE THREAD

**Issue by jmuheim** [keycastr/keycastr#283]
At the moment, when pressing the "Insert" key on an external USB keyboard, results in this:

![CleanShot 2023-11-13 at 18 16 47@2x](https://github.com/keycastr/keycastr/assets/1814983/2d8abea6-e644-4ae9-82f7-00cf5d9d6665)

It seems to be some kind of circle symbol `O`, but interestingly it doesn't really allocate its space when displayed. Pressing "Insert" several times results in exactly the same (no stacking up of several key strokes).

When I press "Insert" plus immediately any other key (for example, "Space"), it looks like this:
  
![CleanShot 2023-11-13 at 18 21 26@2x](https://github.com/keycastr/keycastr/assets/1814983/e2119053-732d-4b8b-9c32-8cb812ede79b)

The following key ("Space") somehow overlaps the first ("Insert") one.

You might wonder: why bother? The "Insert" key is not available on Mac keyboards anyway!

But I'm regularly creating instructional videos for the NVDA screen reader, which runs under Windows (in my case, inside a VMware Fusion virtual machine) and uses "Insert" a lot. So it would be great to have it displayed properly, or at least not totally confused!

On Mac, the "Insert" key apparently maps to the "Help" key (on older Mac keyboards), so it would be great to have at least a proper "?" symbol displayed or something like that. Even greater would be in my use case (obviously) something like "Insert".

I asked for this a few years ago (https://github.com/keycastr/keycastr/issues/74), but never received an update, so I'm trying to push this with a new issue. If I can be of any help (or give a donation), please reach out.

Thank you.

**Comment by akitchen** [comment]
Thank you for bringing this up again. This topic did get pushed down the list a bit. The behavior you've shown looks like some kind of strange bug, so it would be good to get this resolved one way or another.

There is a bit more information in the other issue, and I have found and cleaned up an old Mac keyboard. Development time is limited but I will try to look into this soon.

**Comment by akitchen** [comment]
It turns out that this is a simple bug having to do with how the string which represents the help key is described in the code. 

The desired display string for the help key is actually the result of combining two unicode code units, a question mark and an enclosing circle. The question mark was simply missing in the internal mapping, possibly removed by mistake at some point because it looks funny at first glance. The circle `U+20DD` is an enclosing circle, meaning it doesn't have length on its own, which would explain the strange artifact you are currently seeing in the app.

In other words the fix is a pretty trivial one-character change which will make it into the next release.

**Comment by jmuheim** [comment]
Thank you so much, @akitchen!
